src/components/atoms/AccessDeniedModal/AccessDeniedModal.tsx

```diff
@@ -1,50 +0,0 @@
-import React, { useCallback, useState } from "react";
```

Comment:
removed the component `AccessDeniedModal` as it's not used anywhere.
We should better use `ConfirmationModal`